query/src/sql/statements/statement_insert.rs

```diff
@@ -164,7 +164,11 @@ impl DfInsertStatement {
             value_exprs.push(exprs);
         }
 
-        Ok(InsertInputSource::Expressions(value_exprs))
+        let values = format!("{}", values);
```

Comment:
`values` can be  very large, it's expensive to parse the values string to Expr::Values and use format to convert it into String back.